### DISCUSSION TITLE
How DreamBooth training example for FLUX.1 [dev] GPU usage?

### DISCUSSION THREAD

**Discussion by kimtaehyeong** [huggingface/diffusers#9233] [Q&A]
Hello.
I tried to learn the following README_flux.md for example sample learning.

ex) accelerate launch train_dreambooth_flux.py

However, out of memory has occurred.
How much gpu memory do I need for learning? I'm currently using 1 gpu of A100 80G.

Is there a way to learn with less gpu?

**Comment by Adoni** [comment]
Same error here. Did you solve it?

**Reply by kimtaehyeong** [reply]
No, it hasn't been resolved yet

**Reply by kimtaehyeong** [reply]
Please note that we are dealing with a similar issue here as well.
-> https://github.com/huggingface/diffusers/issues/9278#issuecomment-2395336260

**Reply by Adoni** [reply]
Thanks a lot!

**Comment by Zhu1116** [comment]
I also ran this, and it caused an out-of-memory (OOM) error on a single A800 GPU. But when using 8 A800 GPUs, training worked fine. I asked GPT, and the explanation was: multi-GPU training "distributes" the memory load of model parameters and activations across the GPUs. In single-GPU training, one card can't handle all that memory load, so it crashes.
